.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be used across multiple projects.
```

Comment:
¿Por qué dejáis esto aquí?

Reply:
Estube limpiando el yml generado por circle de comentarios claves que no usaba, pero los que si uso como **orbs** y **python** para indicarlo vi conveniente dejarlos. Los quito o traduzo.

Reply:
Lo que tienes que hacer, si es pertinente, es documentarlos.